pkgs/development/python-modules/pymc3/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, Theano
+, pandas
+, patsy
+, joblib
+, tqdm
+, six
+, h5py
+, run-tests ? false # these are quite heavy
```

Comment:
This should not be an option. If one wants to change this value one can use `overridePythonAttrs`.

Reply:
fixed, thank you